site/cds_rdm/communities/views.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2025 CERN.
+#
+# CDS-RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Community views module."""
+
+from flask import g, redirect, request, url_for
+from invenio_communities.views.communities import (
+    HEADER_PERMISSIONS,
+    render_community_theme_template,
+)
+from invenio_communities.views.decorators import pass_community
+from invenio_rdm_records.proxies import (
+    current_community_records_service,
+    current_rdm_records,
+)
+from invenio_rdm_records.resources.serializers import UIJSONSerializer
+from invenio_records_resources.services.errors import PermissionDeniedError
+
+
+def publications_metric_blr(results):
+    """Publications metric for BLR."""
+    resource_types = results._results.aggregations.resource_types.buckets
+    valid_pubication_types = [
```

Comment:
Can you please check the values here?
```suggestion
    valid_publication_types = [
```